api/namex/services/name_request/name_request_state.py

```diff
@@ -76,18 +76,23 @@ def display_reapply_action(nr_model=None) -> Boolean:
     """Logic for displaying the renew button."""
     try:
         if nr_model and nr_model.stateCd in (State.CONDITIONAL, State.APPROVED):
-            if nr_model.expirationDate and not nr_model.is_expired and not nr_model.has_consumed_name:
-                todays_date = datetime.utcnow().date()
-                expiry_date = nr_model.expirationDate.date()
-
-                delta = expiry_date - todays_date
-                if delta.days <= 14:
-                    return True
+            if not nr_model.is_expired and not nr_model.has_consumed_name:
+                return is_reapplication_eligible(nr_model.expirationDate)
         return False
     except Exception as err:
         raise NameRequestActionError(err)
 
 
+def is_reapplication_eligible(expiration_date) -> Boolean:
+    if expiration_date:
+        todays_date = datetime.utcnow().date()
```

Comment:
`datetime.utcnow()` is depreciated

There's an equivalent - `datetime.now(timezone.utc)` I think